package-lock.json

```diff
@@ -18,13 +414,13 @@
       }
     },
     "balanced-match": {
-      "version": "1.0.0",
-      "resolved": "https://registry.npmjs.org/balanced-match/-/balanced-match-1.0.0.tgz",
-      "integrity": "sha1-ibTRmasr7kneFk6gK4nORi1xt2c="
+      "version": "1.0.2",
+      "resolved": "https://registry.npmjs.org/balanced-match/-/balanced-match-1.0.2.tgz",
+      "integrity": "sha512-3oSeUO0TMV67hN1AmbXsK4yaqU7tjiHlbxRDZOpH0KW9+CeX4bRAaX0Anxt0tx2MrpRpWwQaPwIlISEJhYU5Pw=="
     },
     "bootstrap": {
       "version": "3.4.1",
-      "resolved": "http://localhost:5559/bootstrap/-/bootstrap-3.4.1.tgz",
```

Comment:
This was the culprit, leading to the following error when running `pip install -e .`

```
npm ERR! FetchError: request to http://localhost:5559/bootstrap/-/bootstrap-3.4.1.tgz failed
```